tests/Feature/ElectorTest.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+
+namespace Tests\Feature;
+
+
+use App\Election;
+use App\Elector;
+use App\User;
+use Carbon\Carbon;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+use Tests\TestCase;
+
+class ElectorTest extends TestCase
```

Comment:
Thanks for the tests! It is very helpful for me to see example `factory()` calls.